ui-admin/src/components/forms/TextInput.tsx

```diff
@@ -26,6 +27,7 @@ export const TextInput = (props: TextInputProps) => {
         htmlFor={inputId}
       >
         {label}
+        {required && <span className="text-danger">*</span>}
```

Comment:
I questioned if the `*` should actually be in the label, but it looks like the answer is "yes": https://www.deque.com/blog/anatomy-of-accessible-forms-required-form-fields/#:~:text=Providing%20Star%20(Asterisk)%20Symbol,is%20accessible%20to%20assistive%20technologies.